Add notebook intro about pandas

celinedurniak opened the pull request:
I added a jupyter notebook in 3_using_external_libraries: 5_data_analysis_library_pandas.ipynb

There is also a csv file used for one of the exercises.
Note that to read and write HDF5 and Excel files additional libraries might have to be installed.
I added the instructions to do so as comments.

The content of the notebook is only an introduction to pandas. For example, 'reshaping' is not described and the parts about plotting and grouping could easily be extended.

celinedurniak committed:
Add notebook intro about pandas

wpotrzebowski reviewed (commented):
Impressive work (as always)!
Some comments: 
1. It may be worth adding a few words about motivation for using it (rather than numpy for example)
2. I suppose there is typo in "The representation of _firstseries shows the index on the left and the values on the right"
3. You may want to add comment what "first_series * 2" does.
4. You may consider moving solutions to "solutions" folder

celinedurniak committed:
Corrected pandas notebook after review

celinedurniak commented:
Thanks for the review. I made some modifications:
- solutions moved to solutions folder
- added some advantages of using the library
- corrected typo
